src/auth/identitypoolclient.ts

```diff
@@ -229,6 +229,7 @@ export class IdentityPoolClient extends BaseExternalAccountClient {
       method: 'GET',
       headers,
       responseType: formatType,
+      retry: true,
```

Comment:
Why is this just hardcoded instead of using the RETRY_CONFIG?

Reply:
Good catch! Fixed